crates/apub/src/lib.rs

```diff
@@ -160,13 +160,13 @@ pub(crate) async fn local_site_data_cached(
             Instance::blocklist
           ))?;
 
-        Ok::<_, diesel::result::Error>(Arc::new(LocalSiteData {
+        Ok::<_, LemmyError>(Arc::new(LocalSiteData {
           local_site,
           allowed_instances,
           blocked_instances,
         }))
       })
-      .await?,
+      .await.map_err(|_e| LemmyErrorType::NotFound)?,
```

Comment:
Its generally not a good idea to ignore errors, so use `.with_lemmy_type()` or simply remove map_err.

Reply:
I couldn't find a way around this one.

Reply:
Right this one is strange, in other places it is handled like this:
```rust
    .await
    .map_err(|e| anyhow::anyhow!("err getting activity: {e:?}"))
```

Reply:
Done